### PR TITLE
feat(pci): remove usage of otrs module in favor of the support module

### PR DESCRIPTION
**Pull request by FredericEspiau**
MANAGER-2803
MANAGER-3270

### CODE REVIEW

**Comment by cbourgois** [comment]
Related to https://github.com/ovh-ux/manager/pull/1316

File: packages/manager/apps/public-cloud/src/index.html [hunk 1]
```diff
@@ -34,7 +34,7 @@ <h1 class="mt-3 mb-5" data-ng-if="('preload_manager_ovh' | translate) !== 'prelo
                     </div>
                 </div>
             </div>
-            <div data-otrs-popup></div>
+
```
Comment (marie-j) [inline packages/manager/apps/public-cloud/src/index.html]: This line could be removed

Comment (FredericEspiau) [reply packages/manager/apps/public-cloud/src/index.html]: I like it though, it spaces the code

**Comment by jleveugle** [comment]
Hello @FredericEspiau,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request closed without merging**